app/retail/templates/admin/index.html

```diff
@@ -17,9 +17,13 @@
 
 {% endcomment %}
 
-{% load i18n static %}
+{% load i18n static compress %}
 
-{% block extrastyle %}{{ block.super }}<link rel="stylesheet" type="text/css" href="{% static "admin/css/dashboard.css" %}">{% endblock %}
+{% block extrastyle %}{{ block.super }}
+    {% compress css file admin_dashboard %}
+        <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/admin/dashboard.scss" %} />
+    {% endcompress %}
+{% endblock %}
```

Comment:
re-indent with 2 spaces

Reply:
That is done, and I've fixed another ~20 templates too!